editor/src/components/common/actions/index.ts

```diff
@@ -15,21 +14,22 @@ export type InspectorPanel = 'inspector'
 
 export type CodeEditorPanel = 'uicodeeditor'
 
+export type NavigatorPanel = 'navigatorPane' | 'navigator'
```

Comment:
what is navigator and what is navigatorPane?

Reply:
navigatorPane is the fullheight pane which contains the navigator for now, it can be toggled from the right menu

navigator is part of the navigatorPane, it can be closed/open from the arrow toggle on the topright corner of the navigator (it keeps the pane open, but closes the Elements area)

Reply:
(minor nitpick / soapbox point: that `Navigator` area inside `<Design>` should be a `pane`. One of the `panels` inside it is the existing navigator. Another (in the future) would be the `DetailNavigator`. Rule: `panes` _contain_ `panels`.